test/integration.test.ts

```diff
@@ -7,4 +24,178 @@ describe("integration", () => {
     // expect(this.integrationSetup.fees.address).toBeDefined();
     // expect(this.integrationSetup.snVat.address).toBeDefined();
   });
+
+  it("test deposit", async function () {
+    // dss.dai.mint(address(this), 100 ether);
+    await this.integrationSetup.dss.dai.mint(
```

Comment:
I would eliminate at least `this.integrationSetup` prefix. Optimally `dss`, `snDss` should be available at the module level.